internal/handler/cmd_query.go

```diff
@@ -46,7 +46,9 @@ func (h *Handler) CmdQuery(ctx context.Context, query *wire.OpQuery) (*wire.OpRe
 
 		dbName, err := common.GetRequiredParam[string](document, "db")
 		if err != nil {
-			return nil, lazyerrors.Error(err)
+			reply.SetDocument(replyDoc)
+
+			return reply, nil
```

Comment:
Cannot be tested in integration test because go driver returns `"database name cannot be empty"` before sending request to FerretDB.